lib/manageiq/rpm_build/generate_core.rb

```diff
@@ -148,6 +149,12 @@ def symlink_plugin_paths(source_dir, target_path)
           end
         end
       end
+
+      def cleanse_locale_files
+        # remove all comment lines from locale files
+        # the comment lines beginning with #, are special
+        `sed -i '' -e '/^#[^,]/d' #{miq_dir.join('locale/*/*.po')}`
```

Comment:
```suggestion
        shell_cmd("sed -i '' -e '/^#[^,]/d' #{miq_dir.join('locale/*/*.po')}")
```

Reply:
Also we have to be careful here with sed.  I think the -i works differently on Mac vs Linux.  @kbrock where did you run this command?

Reply:
ran on mac and it worked fine
ran this through the full rpm build process (docker linux) and it ran fine.

the `-i ''` is what I do to make it work better with both.
Pretty sure on linux you do not need the `''`

Reply:
I'm concerned that on linux it will blow up - maybe @bdunne or @agrare can try?

Reply:
`sed -i ''` doesn't seem to work:
```bash
adam@desktop:~/src/manageiq/manageiq$ sed -i '' -e '/^#[^,]/d' locale/en/manageiq.po 
sed: can't read : No such file or directory
adam@desktop:~/src/manageiq/manageiq$ sed -i -e '/^#[^,]/d' locale/*/*.po
```

The `sed(1)` man page says:
>        -i[SUFFIX], --in-place[=SUFFIX]
>            edit files in place (makes backup if SUFFIX supplied)

So I think what you want is actually:
```bash
adam@desktop:~/src/manageiq/manageiq$ sed -i'' -e '/^#[^,]/d' locale/*/*.po 
```

Aka the `''` immediately after the `-i` with no space.

Reply:
no more `sed`